docs/source/tracking.rst

```diff
@@ -72,7 +72,7 @@ There are different kinds of remote tracking URIs:
 - Database encoded as ``<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>``. MLflow supports the dialects ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``. For more details, see `SQLAlchemy database uri <https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls>`_.
 - HTTP server (specified as ``https://my-server:5000``), which is a server hosting an :ref:`MLflow tracking server <tracking_server>`.
 - Databricks workspace (specified as ``databricks`` or as ``databricks://<profileName>``, a `Databricks CLI profile <https://github.com/databricks/databricks-cli#installation>`_.
-  `See docs <http://docs.databricks.com/applications/mlflow/logging-from-outside-databricks.html>`_ on
+  `See docs <http://docs.databricks.com/applications/mlflow/access-hosted-tracking-server.html>`_ on
```

Comment:
Please add link to Azure doc as well. https://docs.microsoft.com/azure/databricks/applications/mlflow/access-hosted-tracking-server